app/scripts/factories/VariantFactory.js

```diff
@@ -636,6 +633,26 @@ angular.module('oncokbApp')
             this.ncitName = ncitName;
             this.synonyms = synonyms || [];
         }
+        function ReviewedData(item, mutation, drugs, citations) {
+            return {
+                gene: _.isUndefined(item.gene) ? item.hugoSymbol : item.gene.hugoSymbol,
+                uuid: item.uuid,
+                mutation: mutation,
+                tumorType: item.cancerType,
+                drugs: drugs,
+                lastReview: item.lastReview,
+                oncogene: _.isUndefined(item.gene) ? item.oncogene : item.gene.oncogene,
+                tsg: _.isUndefined(item.gene) ? item.tsg : item.gene.tsg,
+                truncatingMutations: item.alteration === 'Truncating Mutations',
+                deletion: item.alteration === 'Deletion',
+                amplification: item.alteration === 'Amplification',
+                level: item.levelOfEvidence,
+                description: item.description,
+                solidPropagationLevel: item.solidPropagationLevel,
+                liquidPropagationLevel: item.liquidPropagationLevel,
+                citations: citations
```

Comment:
Why bother to pass the citations in and not populated through getCitations function? Avoid unnecessary calculation?

Reply:
Citations is calculated from returned data like mutations and drugs. It's not an attribute of item.

Reply:
@victoria34 the only place passing in the citations is getting the citations from item.description. If that's the case, we could just add parameter(includeCitation:boolean) and then parse out the citation in the model.

Reply:
Move citations out of ReviewedData